src/VOTables.jl

```diff
@@ -303,21 +311,27 @@ description(tblxml) = @p let
     nodecontent
 end
 
-fieldattrs(tblxml) = @p let
+get_colmetas(tblxml) = @p let
     tblxml
     @aside ns = _namespaces(__)
     _findall("ns:FIELD", __, ns)
     map() do fieldxml
         attrs = @p attributes(fieldxml) |> map(Symbol(nodename(_)) => nodecontent(_)) |> dictionary
-        
+        basetype = TYPE_VO_TO_JL[attrs[:datatype]]
+        jltype = vo2jltype(attrs)
+        typesize = TYPE_VO_TO_NBYTES[attrs[:datatype]]
+        fixwidth = vo2nbytes_fixwidth(attrs)
+        nullvalues = @p let 
+            fieldxml
+            _findall("ns:VALUES", __, ns)
+            filter(v->haskey(v, "null"))
+            map(v -> v["null"])
+            map(txt -> try parse(basetype, txt) catch err nothing end)
+        end
+        nullvalue = isempty(nullvalues) ? nothing : nullvalues[1]
```

Comment:
There should only be a maximum of one `VALUES` element within the `FIELD` (and of course a maximum of one `null` attribute per `VALUES`) - multiple `VALUES` would be illegal VOTable.  If for some reason - unlikely - the input has multiple `VALUES` this code will just pick up the first one rather than bailing out. I didn't see a more elegant/idiomatic way to handle this in Julia/EzXML, but there may well be one - if so please go ahead and rephrase.